src/plugins/api-token.server.ts

```diff
@@ -176,12 +161,6 @@ const apiToken: Plugin = async (context, inject) => {
   let openverseApiToken: string | undefined
   try {
     openverseApiToken = await getApiAccessToken(context)
```

Comment:
Maybe we should leave these lines so that it's clear if we are using the token when fetching or not?

Reply:
I'm not sure. We're already sending the `error('Unable to retrieve API token, clearing existing token', e)` when things fail, which i think would be enough information to debug this token code in the event of a problem.